docker/quick-setup/ee-with-alert-engine/docker-compose.yml

```diff
@@ -101,6 +103,12 @@ services:
       - gravitee_reporters_elasticsearch_endpoints_0=http://elasticsearch:9200
       - gravitee_plugins_path_0=$${gravitee.home}/plugins
       - gravitee_plugins_path_1=$${gravitee.home}/plugins-ext
+      # AE configuration
+      - gravitee_alerts_alert-engine_enabled=true
+      - gravitee_alerts_alert-engine_ws_discovery=true
+      - gravitee_alerts_alert-engine_ws_endpoints[0]=http://host.docker.internal:8072/
+      - gravitee_alerts_alert-engine_ws_security_username=admin
+      - gravitee_alerts_alert-engine_ws_security_password=adminadmin
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:**       - gravitee_alerts_alert-engine_ws_security_password=**********  
[This comment was created by Aqua Pipeline]